events_ingress/src/test/java/com/adobe/aio/event/publish/feign/FeignPublishServiceTestDrive.java

```diff
@@ -28,7 +28,7 @@ public class FeignPublishServiceTestDrive {
 
   public static void main(String[] args) {
     try {
-      Workspace workspace = WorkspaceUtil.getSystemWorkspaceBuilder().build();
```

Comment:
I would favor keeping the same interface/method-name

Reply:
ok I'll revert back the change

Reply:
done